sources/@roots/bud/src/cli/commands/build.ts

```diff
@@ -258,6 +269,11 @@ export class BuildCommand extends Command {
    * Execute command
    */
   public async execute() {
+    if (!_.isUndefined(this.dashboard))
+      process.stdout.write(
+        `the --dashboard and --no-dashboard flags are deprecated and will be removed in a future release.\n`,
+      )
```

Comment:
`--dashboard` doesn't make sense anymore. i deprecated it so that we aren't failing people's CI annoyingly:

```
❯ yarn bud build --dashboard
the --dashboard and --no-dashboard flags are deprecated and will be removed in a future release.
[bud compilation] › ✔  success   [100]  🎉
```